dirty_cat/_gap_encoder.py

```diff
@@ -609,12 +609,40 @@ class GapEncoder(BaseEstimator, TransformerMixin):
         values (NaN) are present during fit (default is to impute).
         In the inverse transform, the missing category will be denoted as None.
 
+    Examples
+    --------
+    >>> enc = GapEncoder(n_components=2)
 
-    Attributes
-    ----------
-    rho_: float
-    fitted_models_: typing.List[GapEncoderColumn]
-    column_names_: typing.List[str]
```

Comment:
These attributes should stay, but they should also be documented like in the `SuperVectorizer`.

Reply:
Maybe this can be addressed in another PR.

Reply:
Ok

Reply:
Done. I agree it's good not to mix up things :)